changelog.md

```diff
@@ -8,6 +8,8 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 - Updated outdated packages.
+- Exposed `response_timeout` parameter in `open_connection` methods.
```

Comment:
"Exposed" indicates that it was already present but hidden. I'd consider moving it to "Added" section and rephrasing to:
> "Added `response_timeout` parameter in `open_connection` methods."